--- a/src/DependencyInjection/TinymceExtension.php
+++ b/src/DependencyInjection/TinymceExtension.php
@@ -7,7 +7,7 @@ use Eckinox\TinymceBundle\Util\TinymceConfigurator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
 use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
 use Symfony\Component\DependencyInjection\Loader;
-use Symfony\Component\HttpKernel\DependencyInjection\Extension;
+use Symfony\Component\DependencyInjection\Extension\Extension;

 /**
  * This is the class that loads and manages your bundle configuration.
